tests/test_accessors.py

```diff
@@ -59,3 +65,35 @@ def test_postprocess(name, cf_grid_mapping_name):
     assert 'PB' not in ds.data_vars
     assert 'PH' not in ds.data_vars
     assert 'PHB' not in ds.data_vars
+
+
+@pytest.mark.parametrize('test_grid', ['lambert_conformal', 'mercator'], indirect=True)
+def test_dataarray_destagger(test_grid):
+    data = test_grid['U']
+    destaggered = data.xwrf.destagger()
+
+    # Check shape reduction and dim name adjustment
+    assert destaggered.sizes['west_east'] == data.sizes['west_east_stag'] - 1
+
+    # Check coordinate reduction
+    xr.testing.assert_allclose(destaggered['XLAT'], test_grid['XLAT'])
```

Comment:
Maybe I'm confused right now, but should this not be checking whether `XLONG` is close? `XLAT` is going to be `north_south`.
```suggestion
    xr.testing.assert_allclose(destaggered['XLONG'], test_grid['XLONG'])
```

Reply:
Either XLONG or XLAT work! They are 2D anyway, so this is really a test of XLAT_U to XLAT. Would it make sense to have both in the test?

Reply:
I think it might be nice to test both, since the difference in `XLAT` will be much smaller (only curvilinear factor) than `XLONG`. In the `met_em_sample` dataset, e.g. the `XLAT` - `XLAT_U` difference is just 7e-6 while the `XLONG` - `XLONG_U` difference is 4e-4. Maybe this is a bit paranoid, but I fear that `XLAT` could slip below the `allclose` tolerance while we have a much more robust test available.

Reply:
Both XLAT and XLONG tests are now in place, and both pass as-is.